dataland-frontend/tests/component/components/resources/frameworkDataSearch/FrameworkDataSearchResults.cy.ts

```diff
@@ -1,11 +1,11 @@
 import FrameworkDataSearchResults from "@/components/resources/frameworkDataSearch/FrameworkDataSearchResults.vue";
 import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
-import { type DataSearchStoredCompany } from "@/utils/SearchCompaniesForFrameworkDataPageDataRequester";
+import {BasicCompanyInformation} from "../../../../../build/clients/backend";
```

Comment:
import statement should be something like "@"

Reply:
fixed

Reply:
ok